src/sync.d

```diff
@@ -579,8 +580,15 @@ final class SyncEngine
 			
 			// Are there any changes to process?
 			if (("value" in changes) != null) {
-				// There are valid changes
-				log.vdebug("Number of changes from OneDrive to process: ", count(changes["value"].array));
+				auto nrChanges = count(changes["value"].array);
+
+
```

Comment:
un-needed extra line here

Reply:
removed one empty line there. I still like to have one remaining between the var def and the rest.